packages/starlight/components.ts

```diff
@@ -1,3 +1,4 @@
+// @ts-nocheck
```

Comment:
[Chatted](https://discord.com/channels/830184174198718474/1070481941863878697/1192244027223973978) with @Princesseuh about this and sounds like we should actually fix this properly by setting `"jsx": "preserve"` in `tsconfig.json`.

```suggestion
```

Reply:
Worth checking whether that leaks into starlight. It might need to become a js file instead. I'll take a look.

Reply:
All good. This file did not error on the user-side in the first place.